velox/functions/lib/TimeUtils.h

```diff
@@ -123,4 +126,155 @@ struct InitSessionTimezone {
     timeZone_ = getTimeZoneFromConfig(config);
   }
 };
+
+FOLLY_ALWAYS_INLINE std::optional<DateTimeUnit> fromDateTimeUnitString(
```

Comment:
Would you please add some comments for the new API? If there are unit tests for them, please move them into the lib; if not, we will need to add them.

Reply:
And maybe mention it in the currently empty PR description/commit message.

Reply:
Added.